pkg/loopback/loop_wrapper.go

```diff
@@ -16,6 +16,9 @@ package loopback // import "github.com/docker/docker/pkg/loopback"
 */
 import "C"
 
+// nolint: deadcode
+// nolint: unused
+// nolint: varcheck
```

Comment:
Apparently this is still not good enough :sob: as structcheck also complains about unused fields.

Also, to make things more compact, maybe use `// nolint: deadcode,unused,varcheck,structcheck`

Reply:
Ah, good one; was actually wondering if they could be combined in a single line 👍